app/src/main/kotlin/com/exallium/todoapp/editnote/EditNotePresenter.kt

```diff
@@ -99,9 +133,15 @@ class EditNotePresenter(view: EditNoteView,
                 }).addToComposite()
     }
 
-    internal fun buildNote(oldNote: Note): Note = model.buildNote(oldNote, view.getNewNoteTitle(), view.getNewNoteBody())
+    internal fun buildNote(oldNote: Note): Note = model.buildNote(oldNote, view.getNoteTitle(), view.getNoteBody())
+
+    internal fun buildNote(): Note {
```

Comment:
I'd make sure internal is the right modifier, internal in kotlin is a little wierd.